CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ endif(IS_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/.git)
 
 find_package(Threads REQUIRED)
 
-find_package(Boost 1.53.0 REQUIRED COMPONENTS program_options filesystem system )
+find_package(Boost 1.63.0 REQUIRED COMPONENTS program_options filesystem system )
```

Comment:
I think that might be a bad idea, we still need to support bionic for @baumeier, which only has boost-1.65: https://packages.ubuntu.com/bionic/libboost-all-dev

Reply:
Ok